cli/terminal.go

```diff
@@ -118,3 +123,100 @@ func ReadPlanFromTerminal() (int, error) {
 	}
 	return planID, nil
 }
+
+// Columns formats a list of strings to a tidy column representation
+func Columns(data []string) (string, error) {
```

Comment:
Is there a need to be public?

Reply:
copied like this from the internal package. Changed because it seams to work.